xml/admin_install_salt.xml

```diff
@@ -1493,8 +1493,8 @@ drive_group_default_name:
     drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
   wal_devices:
     drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
-  block_wal_size: '5G'  # in bytes, you can use shortcuts; optional
-  block_db_size: '5G'   # in bytes, you can use shortcuts; optional
+  block_wal_size: '5G'  # (optional)
+  block_db_size: '5G'   # (optional)
```

Comment:
consider adding something like 'unit suffixes permitted', but if you think it's clear from the example - just ignore this comment